lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -433,6 +424,208 @@ def add_kp_manifest_data(df: pd.DataFrame, manifest_filenames: tuple, manifest_f
     return df.merge(manifest_data[['barcode', 'enrollid']], how='left')
 
 
+@clinical.command("parse-phskc")
+@click.argument("phskc_filename", metavar = "<PHSKC Clinical Data filename>")
+@click.argument("phskc_specimen_manifest_filename", metavar = "<PHSKC Specimen Manifest filename(s)>")
+@click.argument("geocoding_cache_file", metavar = "<Geocoding cache filename>")
```

Comment:
It would be convenient to pass this in as an environment variable if thats ok on your end. Would you mind adding 
` envvar = "GEOCODING_CACHE"` here?